bin/remove-env-var-docs-env

```diff
@@ -0,0 +1,7 @@
+#! /usr/bin/env bash
```

Comment:
question (non-blocking): I don't see this being used anywhere? Seems important to have, but just curious how it's intended to be used? Is it for local development?